features/fork.feature

```diff
@@ -24,6 +24,27 @@ Feature: hub fork
     And "git remote set-url mislav git@github.com:mislav/dotfiles.git" should be run
     And the url for "mislav" should be "git@github.com:mislav/dotfiles.git"
 
+  Scenario: Fork the repository with new remote name specified
+    Given the GitHub API server:
+      """
+      before {
+        halt 400 unless request.env['HTTP_X_ORIGINAL_SCHEME'] == 'https'
+        halt 401 unless request.env['HTTP_AUTHORIZATION'] == 'token OTOKEN'
```

Comment:
This whole `before` block isn't necessary. It was needed for some other scenarios to assert correct authentication, but for this scenario, we don't need to assert those things. You can remove it.